src/model/Stroke.cpp

```diff
@@ -238,6 +238,13 @@ void Stroke::setLastPressure(double pressure) {
     }
 }
 
+void Stroke::setSecondToLastPressure(double pressure) {
+    int pointCount = this->getPointCount();
```

Comment:
Either use `size_t` or auto here, since are working on and STL container and the return type of `getPointCount` should be `size_t`.
I prefer auto here, since it will preserve the `int` until the return type is fixed.

```suggestion
    auto const pointCount = this->getPointCount();
```